package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "my-uix-project",
+  "dependencies": {
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  },
+  "scripts": {
+    "start": "shadow-cljs watch app"
+  },
```

Comment:
would be cool to have prod build cmd here as well

Reply:
@roman01la would these aliases do?
```json
"build": "shadow-cljs compile app",
"release": "shadow-cljs release app"
```

Reply:
yep!

Reply:
what's `"build": "shadow-cljs compile app"` doing? just builds the project once in dev?

I think we only need dev watch build and release prod build

Reply:
not too familiar but i believe that's right:

Compile a build once and exit (dev build? - not sure):
> $ shadow-cljs compile app

Running a release build optimized for production use:
> $ shadow-cljs release app

Reply:
let's keep release and watch ones

Reply:
https://github.com/pitch-io/uix-starter/pull/3/commits/320e9701c0b0c2d1169f8d7fe2ab6a16eb2b6e46 looks ok?

Reply:
perfect!